packages/core/test/optimism/Finder.js

```diff
@@ -0,0 +1,57 @@
+/* External Imports */
+const { web3, ethers } = require("hardhat");
```

Comment:
demonstration script that can be run against `--network optimism` or `--network hardhat`. Things to look at:
- `waffle`: The chai matchers like `expect(tx).to.be.revertedWith("error message")` come with waffle, which is pretty cool.

Reply:
This is cool!